reporter-opentsdb/src/main/java/io/ultrabrew/metrics/reporters/opentsdb/OpenTSDBHttpClient.java

```diff
@@ -77,6 +80,30 @@ void write(final String metricName,
       final String[] tags,
       final long timestamp,
       final String value) throws IOException {
+    // validation
+    if (Strings.isNullOrEmpty(metricName)) {
+      LOG.warn("Null or empty metric name.");
```

Comment:
Same comment as the other similar place, these logs have potential to generate a _lot_ of messages, I wonder if we should have some kind of rate limiting in place?

Reply:
comment up there.